.github/workflows/msvc-ci.yml

```diff
@@ -150,4 +150,271 @@ jobs:
+          }
+
+          foreach ($to_include in "lua.h", "lualib.h", "lauxlib.h", "luaconf.h", "lua.hpp", "luajit.h")
+          {
+            $artifact = Join-Path -Path "${{ env.LUAJIT_SRC_DIR }}" -ChildPath $to_include;
+            
+            Copy-Item -Path $artifact -Destination $luajit_include_dir;
+          }
+
+          foreach ($to_lib in "luajit.lib", "luajit.exp", "lua51.lib", "lua51.exp")
+          {
+            $artifact = Join-Path -Path "${{ env.LUAJIT_SRC_DIR }}" -ChildPath $to_lib;
+            
+            Copy-Item -Path $artifact -Destination $luajit_lib_dir;
+          }
+          
+          $luajit_jit_dir = Join-Path -Path "${{ env.LUAJIT_SRC_DIR }}" -ChildPath "jit";
+
+          Copy-Item -Path $luajit_jit_dir -Destination $luajit_lmod_dir -Filter "*.lua" -Recurse;
+
+          # pkgconfig variables
+          $pkgconfig_luajit_target_dir = $luajit_target_dir.Replace("\", "/")
+          $pkgconfig_luajit_include_dir = $luajit_include_dir.Replace("\", "/")
+          $pkgconfig_luajit_lib_dir = $luajit_lib_dir.Replace("\", "/")
+          $pkgconfig_luajit_lmod_dir = $luajit_lmod_dir.Replace("\", "/")
+          $pkgconfig_luajit_bin_dir = $luajit_bin_dir.Replace("\", "/")
+
+          # Writing pkgconfig files
+
+          foreach ($pkgconfig_filename in "lua51", "luajit")
+          {
+            $pkgconfig_content = "prefix=${pkgconfig_luajit_target_dir}",
+            "exec_prefix=`${prefix}",
+            "libname=${pkgconfig_filename}",
+            "includedir=${pkgconfig_luajit_include_dir}",
+            "libdir=${pkgconfig_luajit_lib_dir}",
+            "INSTALL_LMOD=${pkgconfig_luajit_lmod_dir}",
+            "INSTALL_CMOD=${pkgconfig_luajit_bin_dir}",
+            "",
+            "Name: LuaJIT",
+            "Description: Just-in-time compiler for Lua",
+            "URL: https://luajit.org",
+            "Version: ${{ env.LUAJIT_VERSION }}",
+            "Requires: ",
+            "Libs: -L`${libdir} -l`${libname}",
+            "Cflags: -I`${includedir}";
+
+            $pkgconfig_content = $pkgconfig_content |
+              Join-String -Separator ([System.Environment]::NewLine);
+
+            $pkgconfig_file = Join-Path -Path $luajit_pkgconfig_dir -ChildPath "${pkgconfig_filename}.pc";
+            Set-Content $pkgconfig_file $pkgconfig_content -NoNewLine;
+          }
+
+          $luajit_target_dir_unix = & C:\msys64\usr\bin\cygpath -u "$luajit_target_dir";
+          $luajit_tar_gz = Join-Path -Path "${{ runner.temp }}" -ChildPath "luajit.tar.gz";
+          $luajit_tar_gz_unix = & C:\msys64\usr\bin\cygpath -u "$luajit_tar_gz";
+
+          & C:\msys64\usr\bin\bash -lc "tar -C ""$luajit_target_dir_unix"" -czvf ""$luajit_tar_gz_unix"" .";
+
+          Remove-Item -Path $luajit_target_dir -Recurse;
+
+          Add-Content "${{ github.env }}" "LUAJIT_TAR_GZ=${luajit_tar_gz}";
+          Add-Content "${{ github.env }}" "LUAJIT_TAR_GZ_UNIX=${luajit_tar_gz_unix}";
```

Comment:
> That's a lot of "let's write out own `make install` for LuaJit". Meh. (I'll just assume that there is no easier way to do this since you would already have done that in this case)

yes, I tried to integrate with the most famous github actions for Lua [https://github.com/leafo/gh-actions-lua](https://github.com/leafo/gh-actions-lua), but I didn't succeed. I'll try a bit more with their action, but I know I'll cannot promise anything.

The process to build LuaJIT is pretty simple: just launch a Visual Studio command prompt on luajit/src, and run ```msvcbuild.bat```. The boring part is copying all the artifacts to the GTK directory, and also creating a pkgconfig file ```lua51.pc``` for Lua in the proper directory (gtk / lib / pkgconfig)